test/extensions/config_subscription/grpc/eds_resources_cache_impl_test.cc

```diff
@@ -0,0 +1,325 @@
+#include "source/extensions/config_subscription/grpc/eds_resources_cache_impl.h"
+
+#include "test/mocks/event/mocks.h"
+#include "test/test_common/simulated_time_system.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+using testing::NiceMock;
+
+namespace Envoy {
+namespace Config {
+
+class EdsResourcesCacheImplTest : public testing::Test {
+public:
+  using ResourceType = envoy::config::endpoint::v3::ClusterLoadAssignment;
```

Comment:
I leave this up to you, but just a comment that it wasn't clear to me immediately that ResourceType meant ClusterLoadAssignment.  Instead of this typedef, would a `using envoy::config::endpoint::v3::ClusterLoadAssignment` be better, then just use ClusterLoadAssignment everywhere so it's clearer to the reader?

Reply:
Updated the name to ClusterLoadAssignment as it makes it clearer.